che-plugin-openshift-ext-client/src/main/java/org/eclipse/che/ide/ext/openshift/client/deploy/_new/NewApplicationViewImpl.java

```diff
@@ -226,7 +299,41 @@ public void update(int index, KeyValue object, String value) {
         });
 
 
-        Column<KeyValue, String> valueColumn = new Column<KeyValue, String>(new TextInputCell()) {
+        Column<KeyValue, String> valueColumn = new Column<KeyValue, String>(new TextInputCell() {
```

Comment:
I think It would be better to move TextInputCell to inner class. Because anonymous class in constructor of  anonymous is a little bit strange. WDYT?